hazelcast/src/test/java/com/hazelcast/config/YamlConfigBuilderTest.java

```diff
@@ -1980,7 +1980,9 @@ public void testCacheConfig() {
                 + "        size: 1000\n"
                 + "        max-size-policy: ENTRY_COUNT\n"
                 + "        eviction-policy: LFU\n"
-                + "      merge-policy: com.hazelcast.cache.merge.LatestAccessCacheMergePolicy\n"
+                + "      merge-policy:\n"
+                + "         batch-size: 100\n"
+                + "         class-name: com.hazelcast.cache.merge.LatestAccessCacheMergePolicy\n"
```

Comment:
The class `com.hazelcast.cache.merge.LatestAccessCacheMergePolicy` does not exist any more (but it is probably fine here).

Reply:
fixed.